dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -304,7 +307,6 @@ public void setDirtyBit(boolean dirtyBit) {
     void updateByUser(final Version<?> version) {
         this.getVersionMetadata().hidden = version.isHidden();
         this.setDoiStatus(version.getDoiStatus());
-        this.setDoiURL(version.getDoiURL());
```

Comment:
This code allows user to modify the DOI URL. I don't believe this is used and we probably shouldn't allow the user to edit DOI data so I removed this line